### PR TITLE
bash_completion: bump

### PR DESCRIPTION
**Pull request by JakobDev**
New Version of bash completions. btw. I noticed that bash:completion not getting sourced at start. You have to do this by hand. Maybe we need to fix the bash recipe (which contains the line that should start bash_completion).  I also think we should try to upstream the completions of pkgman and finddir.

### CODE REVIEW

**Comment by diversys** [comment]
Please `git mv` older recipe instead of creating new one.

**Comment by JakobDev** [comment]
@diversys How can I do that when I already opened a PR?

File: app-shells/bash_completion/patches/bash_completion-2.11.patchset [hunk 1]
```diff
@@ -0,0 +1,102 @@
+From c83f6af3e75d67aa7789c5656f872c4b3aaf49cc Mon Sep 17 00:00:00 2001
+From: JakobDev <jakobdev@gmx.de>
```
Comment (Begasus) [inline app-shells/bash_completion/patches/bash_completion-2.11.patchset:2]: Would be nice if the credits were kept as they were (quick look doesn't show any real diffs):
https://github.com/haikuports/haikuports/blob/f85fb2d8ddbc31ce149a3395a48ff1cca61c05c9/app-shells/bash_completion/patches/bash_completion-2.7.patchset#L2

**Comment by diversys** [comment]
> @diversys How can I do that when I already opened a PR?

I think you can just force push.

**Comment by korli** [comment]
> @diversys How can I do that when I already opened a PR?

Or just push a commit which removes the old recipe.

**Pull request merged**